packages/forklift-console-plugin/src/modules/Plans/views/details/components/SettingsSection/components/SharedDisksDetailsItem/utils/helpers.ts

```diff
@@ -0,0 +1,39 @@
+import { OnConfirmHookType } from 'src/modules/Providers/modals';
+
+import { V1beta1Plan } from '@kubev2v/types';
+import { k8sPatch } from '@openshift-console/dynamic-plugin-sdk';
+
+interface EnhancedPlan extends V1beta1Plan {
+  spec: {
+    migrateSharedDisks?: boolean;
+  } & V1beta1Plan['spec'];
+}
+
+export const getMigrateSharedDisks = (resource: V1beta1Plan): boolean => {
+  return (resource as EnhancedPlan)?.spec?.migrateSharedDisks ?? true; // when missing, default behavior is true, follow BE description: https://github.com/kubev2v/forklift/pull/1318#issue-2812305530
```

Comment:
this will always be true

Reply:
This will be false only on cases where it's explicitly state false for `migrateSharedDisks`, when undefined it default to true value according to backend logic